CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.23.0
+
+- feat(publish): Ability to merge to non-default (#245)
```

Comment:
Also note removal of #220?

Reply:
Not sure it is note-worthy as we are not removing a feature. It is just obsolete.